client/my-sites/marketing/do-it-for-me/site-info-collection.tsx

```diff
@@ -39,6 +39,7 @@ function SiteInformationCollection() {
 	return (
 		<ActionPanel>
 			<ActionPanelBody>
+				<button onClick={ onSubmit }>NEXT </button>
```

Comment:
Can you help me understand why we need a new button on this page? Submitting the form should automatically call `onSubmit`.

Reply:
Good catch! This was a small hack to get me to the next page 🤣